tldw_Server_API/app/core/Watchlists/output_enrichment_handler.py

```
@@ -0,0 +1,200 @@
+                    try:
+                        tags = json.loads(data.get("tags_json", "[]") or "[]")
+                    except (TypeError, ValueError, json.JSONDecodeError):
+                        pass
+                    items.append({
+                        "id": data.get("id"),
+                        "title": data.get("title", "Untitled"),
+                        "url": data.get("url", ""),
+                        "summary": data.get("summary", ""),
+                        "tags": tags if isinstance(tags, list) else [],
+                    })
+        except Exception:
```

Ruff (0.15.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)